README.md

```diff
@@ -22,36 +21,53 @@ Once the repo is cloned, you’ll change directory into the newly created *sense
-
-Now to deploy this code to all device(s) in the application just run the command:
+The build will start running, and look something like this:
 
 ```
-git push balena master
+[main]     Successfully built d5f1de77fad3
+[Info]     Uploading images
+[Success]  Successfully uploaded images
+[Success]  Release successfully created!
+[Info]     Release: f4e3925bf7d32226365225e1b7201b90 (id: 89693)
+[Info]     ┌─────────┬────────────┬────────────┐
+[Info]     │ Service │ Image Size │ Build Time │
+[Info]     ├─────────┼────────────┼────────────┤
+[Info]     │ main    │ 205.13 MB  │ 1 second   │
+[Info]     └─────────┴────────────┴────────────┘
+[Info]     Build finished in 7 seconds
+                \
+                 \
+                  \\
+                   \\
+                    >\/7
+                _.-(6'  \
+               (=___._/` \
+                    )  \ |
+                   /   / |
+                  /    > /
+                 j    < _\
+             _.-' :      ``.
+             \ r=._\        `.
+            <`\\_  \         .`-.
+             \ r-7  `-. ._  ' .  `\
+              \`,      `-.`7  7)   )
+               \/         \|  \'  / `-._
+                          ||    .'
+                           \\  (
+                            >\  >
+                        ,.-' >.'
+                       <.'_.''
+                         <'
 ```
 
-You may see a warning that says
->The authenticity of host ‘git.balena-cloud.com (54.165.162.194)’ can’t be established.
-ECDSA key fingerprint is SHA256:NfwmqnKId5cx1RWpebbEuuM87bCJbdyhzRnqFES9Nnw.
-Are you sure you want to continue connecting (yes/no)?
-
-Type `yes` and hit return to accept. Your machine is asking you to verify the SSH host. You should only see this the first time you connect to balena.
-
-It should only take about two minutes to build your code (and subsequent builds will be quicker because of caching). You'll know your code has been successfully compiled and built when our friendly unicorn mascot appears in your terminal:
-
-![unicorn](https://user-images.githubusercontent.com/3144447/31838641-b8fca76c-b5ab-11e7-92ee-dd49a0652bac.png)
-
 This means your code is safely built and stored on our image registry.
 
-Your application will now be downloaded and executed by the device you have connected in your application fleet. This may take a few minutes, and you can watch the status of the download in your application dashboard.  Optionally, you can speed this up for subsequent builds using balena's [container deltas](https://balena.io/docs/learn/deploy/delta/) if you want!
+Your application will now be downloaded and executed by the device you have connected in your application fleet. This may take a few minutes, and you can watch the status of the download in your application dashboard.  Your balenaCloud dashboard should look a bit like this:
```

Comment:
```suggestion
Your container will now be downloaded and run by the device you have provisioned in your fleet. This may take a few minutes, and you can watch the status of the download on the device page of the dashboard. It should look a bit like this:
```